test/utils/test-utils.ts

```diff
@@ -42,6 +42,11 @@ export interface TestingOptions {
      */
     name?: string;
 
+    /**
+     * Url for the database connection.
+     */
+    url?: string;
```

Comment:
Why this change?

Reply:
This change was more about being able to test if. If you want we can also revert it - but generally I tried it out by modifying the `ormconfig.json`, which required me to allow setting a url here.

Reply:
I get that it's for testing, but no test you added uses this.

We technically can't, either, because the mongo server we set up won't have many features related to multiple instances enabled.

Reply:
Yes exactly. If you want I can also remove it again :-)

Reply:
I removed the corresponding commit, so the offending line is gone again :-)